### DISCUSSION TITLE
Generated frontend.h files produces different results on windows and on linux

### DISCUSSION THREAD

**Discussion by 12345swordy** [dlang/dmd#13354] [General]
This is bad, I had to rely on someone else with a Linux machine to generate the frontend header file for me in order to pass certain test suits.

**Comment by Geod24** [comment]
This is best reported as an issue (and already is): https://issues.dlang.org/show_bug.cgi?id=21423

**Reply by 12345swordy** [reply]
That doesn't change the current fact that it is already making contributing to dmd more difficult than necessarily.

**Reply by ljmf00** [reply]
Shouldn't this issue be `blocker` importance?

**Comment by MoonlightSentinel** [comment]
Note that one can also `git apply` the diff generated by the CI (automation would be nice though - maybe possible via Github Actions?)
